unredact/utils/constants.py

```diff
@@ -0,0 +1,39 @@
+"""Constants used throughout the unredact package."""
+
+DEFAULT_FONT = "Times-Roman"
+
+FONTS = {
```

Comment:
Added a constants file so that your `__main__.py` file is filed with logic and not constant values.